.github/workflows/release.yml

```diff
@@ -99,6 +99,8 @@ jobs:
           delete-branch: true
```

Comment:
(Optional)

The review interface won't let me comment on the lines directly, but the `title` and `body` fields slightly above this line don't currently have a `v` prefix before the new SNAPSHOT version. In contrast the "Prepare release v1.2.3" PR title does have a `v` prefix.

This might be intentional due to it being a snapshot version and not something people would use install directly, but mentioning in case not.

Reply:
This is intentional. Thanks for pointing it out though! :)